npm/test-system.js

```diff
@@ -77,7 +78,7 @@ module.exports = function (exit) {
                 return next();
             });
         }
-    ], exit);
+    ], done);
```

Comment:
These changes make the scripts different from the runtime scripts. Why have you renamed `exit` to `done`?